libs/testing/e2e/src/lib/config/playwright-config.ts

```diff
@@ -0,0 +1,54 @@
+import { defineConfig, devices } from '@playwright/test'
+
+interface GlobalConfigParams {
+  webServerUrl: string
+  port?: number
+  command: string
+  cwd?: string
+  timeoutMs?: number
+}
+
+export const createGlobalConfig = ({
+  webServerUrl,
+  port,
+  command,
+  cwd = '../../../',
+  timeoutMs = 5 * 60 * 1000,
+}: GlobalConfigParams) => {
+  return defineConfig({
```

Comment:
Make function call return object instead of running `return` statement

```typescript
const foo = () => {
  return bar(...)
}
// should be
const foo = () => bar(...)
```

Reply:
Addressing in https://github.com/island-is/island.is/pull/16816